### ISSUE TITLE
Video: 'Replay' button is announced as 'Play'

### ISSUE THREAD

**Issue by shanelin1** [eBay/ebayui-core#1440]
# Bug Report

## Description
Once the video has finished, swipe to focus the 'Replay' button.
Note how the element is announced.

Expected Result
The button should be announced as 'Replay', as in this case the video has already been played once.

Actual Result
The button is announced as 'Play'. Visually the icon changes to 'Replay' icon however screen reader user will still hear it as 'Play'

<button class="shaka-small-play-button material-icons-round" aria-label="Play">replay</button>

ebay-video component need to accept new param "a11y-replay-text"

**Comment by agliga** [comment]
Filed a bug on shaka player for this.

**Comment by agliga** [comment]
https://github.com/google/shaka-player/issues/3512
Seems that it will be fixed in the `3.1.2`

**Comment by ianmcburnie** [comment]
Fixed in Shaka's code. Closing.
